test/test_binary_ufuncs.py

```diff
@@ -1471,44 +1480,6 @@ def _reference_implementation(x, mod):
             _reference_implementation(x, mod)
             _reference_implementation(x_nc, mod)
 
-    @onlyCPU
-    @dtypes(torch.float, torch.long)
-    def test_remainder(self, device, dtype):
```

Comment:
Is it really OK to just remove this test and the following one? The new tests do not seem to cover everything that it does.

Reply:
I think the following tests cover:
1. dtype for float and long
2. Tensor remainder to scalar or scalar tensor (positive and negative)
3. Tensor remainder to another tensor (broadcasting)
The only thing left, which I will put into the new test cases, is negative divisor.